boards/FSM/Inc/Io/Io_FlowMeter.h

```diff
@@ -2,15 +2,43 @@
 
 #include <stm32f3xx_hal.h>
 
+/**
+ * Initializes the primary and secondary flow meter
+ */
+void Io_FlowMeter_Init(void);
```

Comment:
Kind of a nit, but I think turning this file into `Io_FlowMeters.h` would be more apt.

`App_FlowMeter` is a generic application-layer library that hands out individual flow meters. So singular is fine. `Io_FlowMeters.` contains stub function for two flow meters: primary and secondary, so plural is more apt.

Reply:
Renamed to `Io_FlowMeters.h`